Makefile

```diff
@@ -1,6 +1,10 @@
 manage = python manage.py
 install = pip install
 
+docker:
+	sed -e 's/-r common.txt//' requirements/dev.txt \
+	  | cat requirements/common.txt - > requirements.txt
```

Comment:
Ooooh.  Does docker have to _copy_ one requirements.txt file to the container?  I think I understand now.

Do you think you can at least add a commit to gitignore the dummy requirements file?